.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -281,7 +281,11 @@ tasks:
       - func: check
 
   - name: test
-    tags: ['required-for-publish', 'run-on-pr']
+    tags: 
+      - required-for-publish
+      - run-on-pr
+      - assigned_to_jira_team_compass_compass
+      - foliage_creates_breaking_commit
```

Comment:
foliage will not open any ticket for this task, instead sends a slack broken commit message. Slack channel is also pulled from mothra config